content/tbb/tbb-37/contents.lr

```diff
@@ -5,6 +5,6 @@ title: I need Tor Browser in a language that's not English.
 seo_slug: need-tor-browser-language-not-english
 ---
 description:
-We want everyone to be able to enjoy Tor Browser in their own language. Tor Browser is now [available in 25 different languages](https://www.torproject.org/download/languages/), and we are working to add more. Want to help us translate? [See here](https://tb-manual.torproject.org/becoming-tor-translator/)
+We want everyone to be able to enjoy Tor Browser in their own language. Tor Browser is now [available in 30 different languages](https://www.torproject.org/download/languages/), and we are working to add more. Want to help us translate? [See here](https://tb-manual.torproject.org/becoming-tor-translator/)
```

Comment:
Use this url: https://community.torproject.org/localization/becoming-tor-translator/

Reply:
And as a suggestion, replace "See here" to something more meaningful like "Want to help us translate? [Become a Tor translator](https://community.torproject.org/localization/becoming-tor-translator/)!"